### PR TITLE
Gwc datadir

### PR DESCRIPTION
**Pull request by pmauduit**
This PR allows the datadirization of GWC:
    
- POM modification (adding a profile)
- debian|RPM package
- Revamp of georchestra header integration

Tested only into jetty (because GWC was not datadirized yet), with and without the -Dgeorchestra.datadir option.

TODO

* [x] Build
* [x] testsuite
* [ ] with `georchestra.datadir` option
* [ ] without `georchestra.datadir` option

### CODE REVIEW

File: geowebcache-webapp/pom.xml [hunk 1]
```diff
@@ -187,4 +195,135 @@
...
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>net.sf.debian-maven</groupId>
+            <artifactId>debian-maven-plugin</artifactId>
+            <version>1.0.6</version>
+            <configuration>
+              <packageName>georchestra-geowebcache</packageName>
+              <packageDescription>GeoWebCache standalone for geOrchestra</packageDescription>
```
Comment (fvanderbiest) [inline geowebcache-webapp/pom.xml]: Standalone GeoWebCache for geOrchestra

File: geowebcache-webapp/pom.xml [hunk 2]
```diff
@@ -187,4 +195,135 @@
...
+      <id>rpmPackage</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-war-plugin</artifactId>
+            <version>2.6</version>
+            <configuration>
+              <classifier>generic</classifier>
+              <packagingExcludes>**/ext/src/**,**/ext/examples/**,**/ext/pkgs/**,**/docs/**,**/openlayers/examples/**,**/openlayers/tests/**,PostTreatment.groovy</packagingExcludes>
```
Comment (fvanderbiest) [inline geowebcache-webapp/pom.xml]: not relevant

File: geowebcache-webapp/src/deb/resources/etc/georchestra/geowebcache/geowebcache.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
+baseUrl=https://georchestra.mydomain.org
```
Comment (fvanderbiest) [inline geowebcache-webapp/src/deb/resources/etc/georchestra/geowebcache/geowebcache.properties]: http by default

**Comment by fvanderbiest** [comment]
Please merge once the above remarks have been addressed. :-)

File: geowebcache-webapp/pom.xml (hunk 1 above)
Comment (pmauduit) [inline geowebcache-webapp/pom.xml]: done

File: geowebcache-webapp/pom.xml (hunk 2 above)
Comment (pmauduit) [inline geowebcache-webapp/pom.xml]: done

File: geowebcache-webapp/src/deb/resources/etc/georchestra/geowebcache/geowebcache.properties (hunk 3 above)
Comment (pmauduit) [inline geowebcache-webapp/src/deb/resources/etc/georchestra/geowebcache/geowebcache.properties]: done

**Pull request merged**